README.md

```diff
@@ -33,15 +33,29 @@ Use `npm run test:feature` to run E2E tests with the Cypress GUI or simply use `
 
 ### Running the linter
 
-Solidus NextJS Frontent uses StandardJS library as style guide, linter and formatter.
+Solidus NextJS Frontent uses prettier-standard library as style guide, linter and formatter.
 It works as a standlone system and doens't need an `.eslintrc` file.
 
 ```
+  npm run format
   npm run lint
-  npm run lint --fix # Automatically fixes all the offences
 ```
 
-**Important:** install the standardjs extension for you editor [here](https://standardjs.com/#are-there-text-editor-plugins)
+**Important:** install the prettier-standard extension for you editor [here](https://marketplace.visualstudio.com/items?itemName=numso.prettier-standard-vscode)
+
+You can also configure your editor to format the current file on save avoiding the linter errors.
+
+Here there is the VSCode example:
```

Comment:
I'd change this line to "This are the steps to configure the linter on VS Code:"